plugins/inputs/cloudwatch/cloudwatch_test.go

```diff
@@ -201,17 +201,19 @@ func TestSelectMetrics(t *testing.T) {
 				Dimensions: []*Dimension{
 					{
 						Name:  "LoadBalancerName",
-						Value: "*",
+						Value: "lb*",
 					},
 					{
 						Name:  "AvailabilityZone",
-						Value: "*",
+						Value: "us-east*",
 					},
 				},
 			},
 		},
 	}
+	err := c.initializeCloudWatch()
 	c.client = &mockSelectMetricsCloudWatchClient{}
+	assert.NoError(t, err)
```

Comment:
Can you please check the error _directly_ after setting `err`, i.e. move this one line up?!